### ISSUE TITLE
Menu Bar - Current page highlight doesnt work with ajax'd content

### ISSUE THREAD

**Issue by TimCadieux** [wet-boew/wet-boew#7957]
If I do this

    `<nav role="navigation" id="wb-sec" 
            class="col-md-3 col-md-pull-9 visible-md visible-lg" typeof="SiteNavigationElement">
    <h2>Section menu</h2>
    <ul class="list-group menu list-unstyled">
    <li>
        <h3>Products and Servicess</h3>
        <ul class="list-group menu list-unstyled">
            <li><a class="list-group-item" href="/en/home">Home</a></li>

            <li><a class="list-group-item" href="#">Products</a></li>
            <li><a class="list-group-item" href="#">Leadership</a></li>
            <li><a class="list-group-item" href="#">Media Analysis</a></li>
            <li><a class="list-group-item" href="#">Word Cloud</a></li>
            <li><a class="list-group-item" href="#">Resources</a></li>
        </ul>
    </li>
    </ul>
        </nav>`

Then my en/home li is highlighted because its the Current page.

If i instead do 
    `
    <nav role="navigation" id="wb-sec" data-ajax-replace="/assets/ajax/vertical-sitemenu-en.html" 
            class="col-md-3 col-md-pull-9 visible-md visible-lg" typeof="SiteNavigationElement">
        </nav>
`
and have the same html inside the vertical menu file, the current page is never highlighted?

**Comment by kodecount** [comment]
I can't reproduce. I think you're talking about nav current highlighting? What is your breadcrumb?

**Comment by kodecount** [comment]
What is the HTML inside `vertical-sitemenu-en.html` ?

**Comment by TimCadieux** [comment]
Yes and it works fine on my horizontal nav as well.  My breadcrumbs aren't implemented yet..

    `<h2>Section menu</h2>
    <ul class="list-group menu list-unstyled">
    <li>
        <h3>Products and Servicess</h3>
        <ul class="list-group menu list-unstyled">
            <li><a class="list-group-item" href="/en/home">Home</a></li>

            <li><a class="list-group-item" href="#">Products</a></li>
            <li><a class="list-group-item" href="#">Leadership</a></li>
            <li><a class="list-group-item" href="#">Media Analysis</a></li>
            <li><a class="list-group-item" href="#">Word Cloud</a></li>
            <li><a class="list-group-item" href="#">Resources</a></li>
        </ul>
    </li>
</ul>`

**Comment by kodecount** [comment]
If I understand the navcurrent plugin it well high the horizontal menu before the secondary so if the link exist in both menu the first menu well highlighted.

So your using: https://wet-boew.github.io/v4.0-ci/theme/content-secmenu-en.html (always forget about this menu).

**Comment by TimCadieux** [comment]
My horizontal menu is for our division and so doesnt have any of my page links in it.  Also, just adding the vertical menu on its own works just fine..its just when i ajax it in that it displays without highlighting current

**Comment by duboisp** [comment]
@TimCadieux  Can you provide some instructions on how we can reproduce your issue by using the current wet-boew version?

**Comment by TimCadieux** [comment]
I haven't been able to find a single example of using the Sec Menu with more than 1 level deep of navigation.  That's all i'm looking for...

[https://wet-boew.github.io/v4.0-ci/theme/content-secmenu-en.html](https://wet-boew.github.io/v4.0-ci/theme/content-secmenu-en.html)

**Comment by rlambert27** [comment]
Here's an example we use:

https://ssl-templates.services.gc.ca/app/cls/WET/gcweb/v4_0_24/cdts/samples/sectionmenu-en.shtml

**Comment by TimCadieux** [comment]
What Template are you using?  Argh, wow, https://wet-boew.github.io/
appears blocked here now...

**Comment by rlambert27** [comment]
It's the CDTS solution, more info here:

http://www.gcpedia.gc.ca/wiki/Centrally_Deployed_Templates_Solution_(CDTS)

**Comment by TimCadieux** [comment]
Ahh, ok..I really don;t want to start all over again...thx though
